indico/web/client/js/react/util/Overridable.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+
+
+function Overridable({children, overrides, id, ...restProps}) {
+    const childProps = children.props;
+    const Override = overrides[id];
+    const newProps = {...restProps, ...childProps};
+
+    console.log('over', Override, restProps);
+
+    return (Override !== undefined) ? (
+        <Override {...newProps} />
```

Comment:
not needed right now, but eventually it'd be nice to support multiple overrides for a single component